week-1/H-numbers/exercise.js

```diff
@@ -1 +1,13 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+var totalNum = numberOfStudents + numberOfMentors;
+var student ="Number of student :" + numberOfStudents;
+var mentor ="Number of mentors :" + numberOfMentors;
+var total ="Total number of students and mentors :" + totalNum;
```

Comment:
Just a minor detail – please make sure to have a space before and after `=`. Consistency is important.

Reply:
noted, thx